### PR TITLE
fix: Roadmap padding in mobile view

### PR DESCRIPTION
**Pull request by akshitarora921**
closes [#4075](https://github.com/amplication/amplication/issues/4075)
- Add padding to roadmap section
- Update line height
image

### CODE REVIEW

**Comment by souravjain540** [comment]
@yuval-hazaz @abrl91 please review

**Comment by souravjain540** [comment]
@abrl91 reminder

File: public/styles/style.css [hunk 1]
```diff
@@ -10058,6 +10057,11 @@ body {
   line-height: 5rem;
   align-items: center;
 }
+@media (max-width: 450px) {
```
Comment (abrl91) [inline public/styles/style.css:10060]: why do we need this?

File: public/styles/style.css [hunk 2]
```diff
@@ -10058,6 +10057,11 @@ body {
...
+  .amplication-base .roadmap-heading {
+    line-height: 3rem;
```
Comment (abrl91) [inline public/styles/style.css:10062]: why do we need this?

**Comment by abrl91** [comment]
@akshitarora921 Thank you.
I added some comments. I don't understand why we need line height, while the issue is about padding.

File: public/styles/style.css (hunk 2 above)
Comment (akshitarora921) [reply public/styles/style.css:10062]: If we don't add this the 'line-height' will be '5 rem' for mobile view also

**Comment by akshitarora921** [comment]
Correct @abrl91, the issue is about padding but line height is high too so I couldn't resist fix that also. Will remove the line height change, and raise different PR, If you say

**Review by abrl91 — APPROVED** [review]

**Pull request merged**